static/css/ux-library.css

```diff
@@ -0,0 +1,580 @@
+    --teal-700: #00796B;
+    --teal-800: #00695C;
+    --teal-900: #004D3F;
+    --teal-A100: #A7FFEB;
+    --teal-A200: #64FFDA;
+    --teal-A400: #1DE9B6;
+    --teal-A700: #00BFA5;
+
+    --deep-orange-50: #FBE9E6;
+    --deep-orange-100: #FFCCBC;
+    --deep-orange-200: #FFAB91;
+    --deep-orange-300: #FF8A65;
+    --deep-orange-400: #FF7043;
+    --deep-orange-500:  #FF5722;
+    --deep-orange-600: #F4511E;
+    --deep-orange-700: #E64A19;
+    --deep-orange-800: #D94214;
+    --deep-orange-900: #BF360D;
+    --deep-orange-A100: #FF9E80;
+    --deep-orange-A200: #FF6E40;
+    --deep-orange-A400: #FF3D00;
+    --deep-orange-A700: #DD2C00;
+
+    --gray-50: #FAFAFA;
+    --gray-100: #F5F5F5;
+    --gray-200: #EEEEEE;
+    --gray-300: #E0E0E0;
+    --gray-400: #BDBDBD;
+    --gray-500: #9E9E9E;
+    --gray-600: #757575;
+    --gray-700: #757575;
+    --gray-800: #424242;
+    --gray-900: #212121;
+}
+
+p{
+    font-family: var(--main-font-family);
+    font-weight: 400;
+    font-size: 16px;
+    letter-spacing: 0.03px;
+}
+
+h1{
+    font-family: var(--secondary-font-family);
+    font-weight: 300;
+    font-size: 96px;
+    letter-spacing: -1.44px;
+}
+
+h2{
+    font-family: var(--main-font-family);
+    font-weight: 300;
+    font-size: 60px;
+    letter-spacing: -0.3px;
+}
+
+h3{
+    font-family: var(--main-font-family);
+    font-weight: 300;
+    font-size: 48px;
+    margin-bottom: 30px;
+}
+
+h4{
+    font-family: var(--main-font-family);
+    font-weight: 700;
+    font-size: 34px;
+    letter-spacing: 0.085px;
+}
+
+h5{
+    font-family: var(--main-font-family);
+    font-weight: 400;
+    font-size: 24px;
+}
+
+h6{
+    font-family: var(--secondary-font-family);
+    font-weight: 500;
+    font-size: 20px;
+    letter-spacing: 0.03px;
+    margin-bottom: 30px;
+}
+
+.subtitle-1, .leading-text-1{
+    font-family: var(--main-font-family);
+    font-weight: 500;
+    font-size: 16px;
+    letter-spacing: 0.03px;
+}
+
+.subtitle-2, .leading-text-2{
+    font-family: var(--main-font-family);
+    font-weight: 400;
+    font-size: 14px;
+    letter-spacing: 0.014px;
+}
+
+.button-text{
+    font-family: var(--main-font-family);
+    font-weight: 500;
+    font-size: 14px;
+    text-transform: uppercase;
+    letter-spacing: 0.024px;
+}
+
+.caption{
+    font-family: var(--main-font-family);
+    font-weight: 400;
+    font-size: 12px;
+    letter-spacing: 0.048px;
+}
+
+.overline{
+    font-family: var(--main-font-family);
+    font-weight: 400;
+    font-size: 10px;
+    text-transform: uppercase;
+    letter-spacing: 0.15px;
+}
+
+/*UI Components*/
+/*Buttons*/
+
+.disabled{
+    opacity: 0.6;
+    cursor: not-allowed;
+    pointer-events: none;
+  }
+
+.button-sm{
+    width: 75px;
+    height: 51px;
+    flex-shrink: 0;
+    border: none;
+}
+
+.button-med{
+    width: 108px;
+    height: 51px;
+    flex-shrink: 0;
+    border: none;
+}
+
+.button-sm:hover, .button-med:hover{
+    background-color: #000000;
```

Comment:
use variable

(this comment applies to all other hardcoded styles that are still left in this file)

Reply:
Converted remaining colours to variables